afterpay/src/main/res/values/strings.xml

```diff
@@ -19,5 +19,12 @@
     <string name="afterpay_price_breakdown_no_configuration_description" translatable="true">Or pay with After pay</string>
     <string name="afterpay_price_breakdown_info_link" translatable="false">Info</string>
 
+    <string name="afterpay_price_breakdown_intro_none" translatable="false"></string>
+    <string name="afterpay_price_breakdown_intro_make" translatable="false">Make\u0020</string>
+    <string name="afterpay_price_breakdown_intro_pay" translatable="false">Pay\u0020</string>
+    <string name="afterpay_price_breakdown_intro_in" translatable="false">In\u0020</string>
+    <string name="afterpay_price_breakdown_intro_or" translatable="false">Or\u0020</string>
+    <string name="afterpay_price_breakdown_intro_pay_in" translatable="false">Pay In\u0020</string>
```

Comment:
nit: Perhaps move the space to the format string rather than repeating it in each of the options.

Reply:
I think it's a good point, but if the `NONE` option was set, there would be a space at the beginning of the sentence.

Reply:
Yep, missed that one. Good point.